### PR TITLE
PLANNER-1963: [Tasks Assignment] Add the ability of getting the client password from the kie-server keystore

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
looks like a git error
14:15:19 	at hudson.model.Executor.run(Executor.java:429)
14:15:19 Caused by: java.io.IOException: Server returned HTTP response code: 500 for URL: https://raw.githubusercontent.com/kiegroup/droolsjbpm-build-bootstrap/master/script/repository-list.txt
14:15:19 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1900)

**Comment by wmedvede** [comment]
jenkins retest this

**Comment by wmedvede** [comment]
failed in non related tests
org.kie.camel.embedded.camel.component.CamelEndpointWithJaxWrapperCollectionTest.testWorkingSetGlobalTestSessionSetAndGetGlobal
org.kie.camel.embedded.camel.component.JSonBatchExecutionTest.testManualFireAllRules
org.kie.camel.embedded.camel.component.JSonBatchExecutionTest.testInsertElements
org.kie.camel.embedded.camel.component.JSonBatchExecutionTest.testInsertElementsWithReturnObjects
org.kie.camel.embedded.camel.component.XStreamBatchExecutionTest.testInsertElementsWithReturnObjects

**Comment by wmedvede** [comment]
jenkins retest this

**Comment by wmedvede** [comment]
Hi @rsynek would you mind have a look at this quick improvement?
Thanks!

**Comment by wmedvede** [comment]
jenkins execute full downstream build

**Comment by wmedvede** [comment]
full build failed in non related tests but artifacts were generated well

Test Result (1 failure / -2)
org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT.org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT

@rsynek would you mind move this forward?

**Comment by rsynek** [comment]
@wmedvede please take a look at the FDB. This is not about a single test failure; the build got stuck and failed on a timeout. I can see a daily build passed 3 days ago (that's the last run), so it doesn't seem the master branch has been broken.

**Comment by wmedvede** [comment]
Let's re-launch it and jump to the next non related error

**Comment by wmedvede** [comment]
jenkins execute full downstream build

File: kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-planning/src/main/java/org/kie/server/services/taskassigning/planning/TaskAssigningConstants.java [hunk 1]
```diff
@@ -36,6 +36,18 @@ private TaskAssigningConstants() {
...
 
+    /**
+     * Property for configuring the alias of the user password for connecting with the jBPM runtime in cases where the
+     * kie-server key store is used.
+     */
+    public static final String JBPM_TASK_ASSIGNING_KEY_STORE_PROCESS_RUNTIME_ALIAS = "kie.keystore.key.taskAssigning.processRuntime.alias";
+
+    /**
+     * Property for configuring the password for getting the key stored under the alias name
+     * JBPM_TASK_ASSIGNING_KEY_STORE_PROCESS_RUNTIME_PWD_ALIAS in cases where the kie-server key store is used.
```
Comment (rsynek) [inline kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-planning/src/main/java/org/kie/server/services/taskassigning/planning/TaskAssigningConstants.java:47]: A typo? If this should refer to the constant defined above, there is extra 'PWD' in the name of the constant.

**Pull request merged**